apps/api/v2/src/ee/bookings/2024-08-13/controllers/bookings.controller.ts

```diff
@@ -542,4 +543,19 @@ export class BookingsController_2024_08_13 {
       data: bookingReferences,
     };
   }
+
+  @Get("/:bookingUid/sessions")
+  @HttpCode(HttpStatus.OK)
+  @Permissions([BOOKING_READ])
+  @UseGuards(ApiAuthGuard, BookingUidGuard)
+  @ApiHeader(API_KEY_OR_ACCESS_TOKEN_HEADER)
+  @ApiOperation({ summary: "Get Video Meeting Sessions. Only supported for Cal Video" })
```

Comment:
Similar to other endpoints, we could update the descrioption to mention explicit passing of version header is required

    description: `<Note>Please make sure to pass in the cal-api-version header value as mentioned in the Headers section. Not passing the correct value will default to an older version of this endpoint.</Note>`,